src/components/vars.vue

```diff
@@ -1,65 +1,88 @@
+          >
+            <span class="mdi mdi-check-circle"></span> using secret from
+            {{ val.matchingSecretParentPath }}
           </div>
-          <div class="w-2/12">
-            <span>{{ val.secretVar }}</span>
+          <div v-else class="text-sm text-red-600">
+            <span class="mdi mdi-alert-circle"></span> no matching secret
           </div>
-          <div class="w-8/12">
-            <div v-if="val.when">
-              <div
-                v-for="whenCondition in getWhenConditions(val.when)"
-                :key="whenCondition.condType"
-              >
-                <div v-if="whenCondition.cond">
-                  <div class="flex">
-                    <div class="w-1/3">
-                      <span>{{ whenCondition.condType }}</span>
-                    </div>
-                    <div class="w-1/3">
-                      <div
-                        v-for="include in whenCondition.cond.include"
-                        :key="include.match"
-                      >
-                        <div>{{ include.match }}</div>
+        </div>
+        <div class="col-span-1">
+          <span>{{ val.secretVar }}</span>
+        </div>
+        <div class="col-span-3">
+          <div class="col-span-1" v-if="val.when">
+            <div
+              v-for="whenCondition in getWhenConditions(val.when)"
+              :key="whenCondition.condType"
+            >
+              <div class="flex grid grid-cols-3 gap-4">
+                <div
+                  v-if="whenCondition.condType && whenCondition.cond"
+                  class="col-span-1"
+                >
+                  <span class="text-ellipsis overflow-hidden">{{
+                    whenCondition.condType
+                  }}</span>
+                </div>
+                <div class="col-span-1">
+                  <div v-if="whenCondition.cond && whenCondition.cond.include">
+                    <div
+                      v-for="include in whenCondition.cond.include"
+                      :key="include.match"
+                    >
+                      <div class="text-ellipsis overflow-hidden">
+                        {{ include.match }}
                       </div>
                     </div>
-                    <div class="w-1/3">
-                      <div
-                        v-for="exclude in whenCondition.cond.exclude"
-                        :key="exclude.match"
-                      >
-                        <div>{{ exclude.match }}</div>
+                  </div>
+                </div>
+                <div class="col-span-1">
+                  <div v-if="whenCondition.cond && whenCondition.cond.exclude">
+                    <div
+                      v-for="exclude in whenCondition.cond.exclude"
+                      :key="exclude.match"
+                    >
+                      <div class="text-ellipsis overflow-hidden">
+                        {{ exclude.match }}
                       </div>
```

Comment:
Are this changes really related to this PR. Looks like a restying using grid and other things. Should be split in another PR to declutter this one.

Reply:
Should I create a separate pr about this changes?

Reply:
If these and other changes are not related to this PR but a restyling that can be split and/or a preparatory change then open new PRs to be merged before this PR.